### PR TITLE
add validation for review comment.

### PR DESCRIPTION
**Pull request by Alice1319**
releated #2202 

image

### CODE REVIEW

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ReviewServiceTemplateRequest.java [hunk 1]
```diff
@@ -22,6 +22,7 @@ public class ReviewServiceTemplateRequest {
     @Schema(description = "The result of review registration.")
     private ServiceReviewResult reviewResult;
 
+    @NotNull
     @Schema(description = "The comment of review registration.")
```
Comment (baixinsui) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ReviewServiceTemplateRequest.java:26]: Just use @NotNull and @NotBlank, the new exception `ReviewCommentValueMissingException` is unnecessary.

File: modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java [hunk 2]
```diff
@@ -493,6 +494,10 @@ public void reviewServiceTemplateRequest(UUID requestId, ReviewServiceTemplateRe
             throw new ReviewServiceTemplateRequestNotAllowed(
                     "Service template request is not allowed to be reviewed.");
         }
+        if (StringUtils.isEmpty(review.getReviewComment())) {
+            throw new ReviewCommentValueMissingException(
```
Comment (baixinsui) [inline modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java:498]: This is unnecessary. and please set the filed `REVIEW_COMMENT` in `ServiceTemplateRequestHistoryEntity`  to nullable=false

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ReviewServiceTemplateRequest.java (hunk 1 above)
Comment (Alice1319) [reply modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ReviewServiceTemplateRequest.java:28]: updated.

File: modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java (hunk 2 above)
Comment (Alice1319) [reply modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java]: removed and updated.

**Comment by wuzhuanhong** [comment]
LGTM

**Comment by profoundwu** [comment]
LGTM

**Comment by WangLiNaruto** [comment]
LGTM

**Comment by luoping-12345** [comment]
LGTM

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**